assets/scripts/script.js

```diff
@@ -1,14 +1,5 @@
 (function ($) {
   $(document).ready(function () {
-
-    $('.disclaimer-overlay').on('click', function () {
-      $tab = $('.woocommerce-tabs ul.tabs li.common_tab_tab a');
-      $tab.trigger('click');
-      $('html, body').animate({
-        scrollTop: $tab.offset().top - 230
-      }, 600);
-    });
-
```

Comment:
@LucaPipolo Do you remember what this code is for? Can't really make sense of it and it doesn't really seem required?

Reply:
@colourgarden, no sorry. I'm quite sure this was coming from the old itstrategen plugin but not sure which was the purpose here. 😖

Reply:
Well as there won't be a `.disclaimer-overlay` in the DOM any longer, it won't have any effect anyway. It looks like it scrolls the page when the lightbox but I don't really understand the purpose of that anyway.